python/Ganga/Utility/execute.py

```diff
@@ -92,7 +93,7 @@ def __reader(pipes, output_ns, output_var, require_output):
     os.close(pipes[1])
     with os.fdopen(pipes[0], 'rb') as read_file:
         try:
-            output_ns.update({output_var: pickle.load(read_file)})
+            output_ns[output_var] = deepcopy(pickle.load(read_file))
```

Comment:
So from the comment in the main thread, I would say we go for two options:

1. Remove deepcopy and see if the bug happens again
2. Keep it in but put a comment saying this fixed a bug that couldn't be found, remove at your own risk 😄